osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/resources/db/migration/V20200408162500000__Additional_device_fields.sql

```diff
@@ -0,0 +1,17 @@
+DO $$
+BEGIN
+
+  IF NOT EXISTS(SELECT 1 FROM information_schema.columns WHERE table_schema = current_schema AND table_name = 'iec60870_device' AND column_name = 'device_type')
+  THEN
+  	ALTER TABLE iec60870_device ADD COLUMN device_type VARCHAR(255) NOT NULL DEFAULT 'DA_DEVICE';
+  	ALTER TABLE iec60870_device ADD COLUMN gateway_device_identification VARCHAR(40);
+  	ALTER TABLE iec60870_device ADD COLUMN information_object_address INTEGER;
```

Comment:
Added additional unique and check constraints. 
Was wondering if it would make sense to include device_type as well in the check? 
(for now, only for device type 'LIGHT_MEASUREMENT_DEVICE' both gateway_device_identification and information_object_address should be set, otherwise both should be null)